internal/drivers/basic/account_management_test.go

```diff
@@ -0,0 +1,118 @@
+/******************************************************************************
+*
+*  Copyright 2024 SAP SE
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*
+******************************************************************************/
+
+package basic
+
+import (
+	"testing"
+	"time"
+
+	keppelv1 "github.com/sapcc/keppel/internal/api/keppel"
+	"github.com/sapcc/keppel/internal/keppel"
+
+	"github.com/sapcc/go-bits/assert"
+)
+
+func TestAccountManagementDriver(t *testing.T) {
+	driver := AccountManagementDriver{
+		skipConfigLoading: true,
+	}
+
+	driver.config.Accounts = []Accounts{
```

Comment:
I would find it nicer to exercise more of the driver code by loading this from a file in a `fixtures` directory.